packages/common-all/src/utils/index.ts

```diff
@@ -872,7 +872,7 @@ export class ConfigUtils {
       opts.note.config.global &&
       !_.isUndefined(opts.note.config.global.enableChildLinks)
     ) {
-      return opts.note.config.global.enableChildLinks;
+      return opts.note.config.global.enableChildLinks!;
```

Comment:
what version of typescript are you using? this should be auto-inferred by typescript with the last line: `!_.isUndefined(opts.note.config.global.enableChildLinks)`

Reply:
I installed the `package.json` as expected, so `4.5.5`